init/relabel-rw.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash --
+set -eu
+if [ /.qubes-relabeled -nt /rw/.autorelabel ]; then
```

Comment:
Theoretically this check is not enough, in case you switch between templates (for example switch from F41 back to F40). For that it would need to compare some hash on the policy or similar, not just the freshness. But in practice, I don't expect issues (most labels that change on updates are outside of /rw).